<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeLine</class>
 <widget class="QWidget" name="TimeLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
   <item>
    <widget class="omm::Slider" name="slider" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QSpinBox" name="sp_min">
       <property name="minimum">
        <number>-99999999</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="value">
        <number>-11</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sp_value">
       <property name="minimum">
        <number>-99999999</number>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
       <property name="value">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sp_max">
       <property name="maximum">
        <number>99999999</number>
       </property>
       <property name="value">
        <number>111</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_play">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_reset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omm::Slider</class>
   <extends>QWidget</extends>
   <header>managers/timeline/slider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
